### ISSUE TITLE
Grid has multiple <tbody> elements in the same table

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1851]
### Description of the bug

Found by SSBs UU testing 🥇 
Each row in our grid has a `tbody`. Based on HTML documentation a `tbody` is the main content of a table.

> The <tbody> HTML element encapsulates a set of table rows (<tr> elements), indicating that they comprise the body of a table's (main) data.

You can have multiple `tbody`but they would typicly have their own headers. Indicating different sections in the table. 

- [ ] Basic tables should only have one `tbody`
- [ ] How flexible should our grids be? Are there a need to build complex tables with multiple `tbody`?

### Steps To Reproduce

Go to a grid and inspect it
See that all rows have their own `tbody`

**Comment by olemartinorg** [comment]
Fixed in #2513, but this appears in the console now:

```Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.```

:cry: 

See this for more info: https://github.com/facebook/react/issues/5652#issuecomment-164175731

What do you think, @adamhaeger @Magnusrm?

**Comment by adamhaeger** [comment]
> Fixed in [#2513](https://github.com/Altinn/app-frontend-react/pull/2513), but this appears in the console now:
> 
> `Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.`
> 
> 😢
> 
> See this for more info: [facebook/react#5652 (comment)](https://github.com/facebook/react/issues/5652#issuecomment-164175731)
> 
> What do you think, [@adamhaeger](https://github.com/adamhaeger) [@Magnusrm](https://github.com/Magnusrm)?

Thanks for testing, update PR here: https://github.com/Altinn/app-frontend-react/pull/2767/files

**Comment by adamhaeger** [comment]
As this was more complicated than I realized I suggest setting this to blocked pending our table investigation.
